.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: php
+sudo: false
+php:
+  - '7.1'
```

Comment:
In composer You require php 7.0 and above so you should run tests for php 7, 7.1 and nightly

Reply:
you're right